### ISSUE TITLE
ModuleNotFoundError: No module named 'tensorflow.models'

### ISSUE THREAD

**Issue by ucasiggcas** [tensorflow/tensorflow#46306]
hi,dear all friends
when I run the [cifar rp](https://github.com/tensorflow/tensorflow/blob/r0.7/tensorflow/models/image/cifar10/cifar10_train.py),
I find the problem down
could you please help me ?

**Comment by ucasiggcas** [comment]
some env
tf-1.15
cuda 10.1
nvidia 418.87
OS ubuntu 16.08

**Comment by adi1220** [comment]
try reinstalling tensorflow.
try : pip install tensorflow --ignore-installed --user

**Comment by Saduf2019** [comment]
@ucasiggcas 
Is there any particular reason for using old version of tf, we have 2.x later versions can you please upgrade and let us know.

**Comment by ucasiggcas** [comment]
some function not in tf2.x?
what should I do ?

**Comment by ucasiggcas** [comment]
such as tf.placeholder,tf.Session

**Comment by ucasiggcas** [comment]
```
>>> import tensorflow as tf
2021-01-11 15:22:58.507388: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
>>> tf.models
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'tensorflow' has no attribute 'models'
>>> tf.__version__
'2.3.0'

```

**Comment by Saduf2019** [comment]
@ucasiggcas 
Can you please try the below code instead and let us know, please refer to [this gist](https://colab.research.google.com/gist/Saduf2019/c45816d2a93c909fa7efe71fd486ea4f/untitled493.ipynb).
```import tensorflow as tf
tf.keras.models.Model()```

**Comment by ucasiggcas** [comment]
hi,I have the same situation as your codes
```
>>> from tensorflow.models.image.cifar10 import cifar10
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'tensorflow.models'
```
```
>>> tf.keras.models.Model()
<tensorflow.python.keras.engine.training.Model object at 0x7fb383052b90>
```

**Comment by bguetarni** [comment]
`tensorflow.models` is not available for version 1.15 (actually it's no longer available from Tensorflow 1).
Then link that you shared ([cifar](https://github.com/tensorflow/tensorflow/blob/r0.7/tensorflow/models/image/cifar10/cifar10_train.py)) seems to be in version r0.7 (according to the branch name).

That's not the case of `tensorflow.keras.models`; its's still available for the latest version of tensorflow.
If you want to use  `tensorflow.models`, downgrade Tensorflow to version r0.7.
Also note that you'll probably have to install it from source, I don't think it's still available in pip or conda.

**Comment by ucasiggcas** [comment]
thx
I know the answer
will try later
